### ISSUE TITLE
Missing file Views/Students/Index.cshtm

### ISSUE THREAD

**Issue by PSCRIMS** [aspnet/Docs#11264]
# Before you open an issue

If the issue is with an ASP.NET Core document, open the issue with the **Content feedback** button at the bottom of that document page.

If the issue is:

- A simple typo or similar correction, you can submit a PR. See [the contributor guide](https://docs.microsoft.com/contribute/#quick-edits-to-existing-documents) for instructions.
- A general support question, consider asking on a support forum site.
- A site design concern, create an issue at [MicrosoftDocs/Feedback](https://github.com/MicrosoftDocs/Feedback/issues/new/choose).
- A problem completing a tutorial, compare your code with the completed sample.
- A duplicate of an open or closed issue, leave a comment on that issue.

# Issue description

{ description here }

# Software versions

Check the .NET target framework(s) being used, and include the version number(s).

- [x] .NET Core
- [ ] .NET Framework
- [ ] .NET Standard

If using the .NET Core SDK, include `dotnet --info` output. If using .NET Framework without the .NET Core SDK, include info from Visual Studio's **Help** > **About Microsoft Visual Studio** dialog.

<details>
<summary><strong>dotnet --info output</strong> or <strong>About VS info</strong></summary>

```console
<replace>
```
</details>

**Comment by PSCRIMS** [comment]
I am following the tutorial and run the scaffolding to add the MVC Controller with Views using Entity Framework. I filled out the fields according to the tutorial
o	In Model class select Student.
o	In Data context class select SchoolContext.
o	Accept the default StudentsController as the name.
o	Click Add.

I am missing the file Views/Students/Index.cshtml  in my project and if I try to create the View under Views/Students Visual Studio responds by telling that Index,cshtml already exists (it does under Views/Home.)  Should I create it as a file in the Views/Students folder and name it Index,cshtml or did I do something wrong when I performed the scaffolding, if this is the case do I go back to the beginning and recreate the project?
